modules/primer-buttons/lib/button.scss

```diff
@@ -181,12 +181,14 @@
   display: inline-block;
   padding: 0;
   font-size: inherit;
+  font-weight: $font-weight-normal;
```

Comment:
We should take these two style changes out since for now we're leaving `btn-link` to work as it is, at least for the time being 😬

Reply:
Thought you might say that! I'll revert that commit then 🙌